plugins/emc/emc_vnx_file_

```diff
@@ -0,0 +1,516 @@
+			${server}_highw_hits.label High Watermark Hits
+			${server}_loww_hits.label Low Watermark Hits
+			${server}_loww_hits.draw STACK
+			${server}_w_hits.label Watermark Hits
+			${server}_hits.label Hits
+			${server}_lookups.label Lookups
+				
+			multigraph emc_vnx_fileresolve
+			graph_title EMC VNX 5300 FileResolve
+			graph_vlabel Entries
+			graph_category memory
+			graph_args --base 1000
+			${server}_dropped.label Dropped Entries
+			${server}_max.label Max Limit
+			${server}_used.label Used Entries
+			EOF
+			fi
+		if [ $STATSTYPE = "NFS" ] ; then
+#nfs.v3.op data
+			member_elements=$($SSH server_stats $server -info nfs.v3.op | grep member_elements | sed -ne 's/^.*= //p')
+			IFS=',' read -ra graphs <<< "$member_elements"
+			cat <<-EOF 
+			multigraph vnx_emc_v3_calls_s
+			graph_title EMC VNX 5300 NFSv3 Calls per second
+			graph_vlabel Calls
+			graph_category nfs
+			graph_args --base 1000
+			EOF
+			for graph in "${graphs[@]}"; do
+				field=$(echo "$graph" | cut -d '.' -f4 )
+				echo "${server}_$field.label $server $field"
+			done
+
+			cat <<-EOF
+			
+			multigraph vnx_emc_v3_usec_call
+			graph_title EMC VNX 5300 NFSv3 uSeconds per call
+			graph_vlabel uSec / call
+			graph_category nfs
+			graph_args --base 1000
+			EOF
+			for graph in "${graphs[@]}"; do
+				field=$(echo "$graph" | cut -d '.' -f4 )
+				echo "${server}_$field.label $server $field"
+			done
+			cat <<-EOF
+
+			multigraph vnx_emc_v3_op_percent
+			graph_title EMC VNX 5300 NFSv3 Op %
+			graph_vlabel %
+			graph_scale no
+			graph_category nfs
+			EOF
+			for graph in "${graphs[@]}"; do
+				field=$(echo "$graph" | cut -d '.' -f4 )
+				echo "${server}_$field.label $server $field"
+				echo "${server}_$field.min 0"
+			done
+			graphs=()
+#nfs.v4.op data
+			member_elements=$($SSH server_stats $server -info nfs.v4.op | grep member_elements | sed -ne 's/^.*= //p')
+			IFS=',' read -ra graphs <<< "$member_elements"
+			cat <<-EOF
+			multigraph vnx_emc_v4_calls_s
+			graph_title EMC VNX 5300 NFSv4 Calls per second
+			graph_vlabel Calls
+			graph_category nfs
+			graph_args --base 1000
+			EOF
+			for graph in "${graphs[@]}"; do
+				field=$(echo "$graph" | cut -d '.' -f4 )
+				echo "${server}_$field.label $server $field"
+			done
+
+			cat <<-EOF
+
+			multigraph vnx_emc_v4_usec_call
+			graph_title EMC VNX 5300 NFSv4 uSeconds per call
+			graph_vlabel uSec / call
+			graph_category nfs
+			graph_args --base 1000
+			EOF
+			for graph in "${graphs[@]}"; do
+				field=$(echo "$graph" | cut -d '.' -f4 )
+				echo "${server}_$field.label $server $field"
+			done
+			cat <<-EOF
+			
+			multigraph vnx_emc_v4_op_percent
+			graph_title EMC VNX 5300 NFSv4 Op %
+			graph_vlabel %
+			graph_scale no
+			graph_category nfs
+			EOF
+			for graph in "${graphs[@]}"; do
+				field=$(echo "$graph" | cut -d '.' -f4 )
+				echo "${server}_$field.label $server $field"
+				echo "${server}_$field.min 0"
+			done
+
+#nfs.client data
```

Comment:
Yes, I think  comments should usually be indented just like the code they belong to.
(the same goes at least for line 234 and 275, too)
But this is just a matter of style: proceed as you wish.